packages/bot/src/stores/journal-store.js

```diff
@@ -48,9 +48,9 @@ export default class JournalStore {
         const time = formatDate(this.serverTime.get(), 'HH:mm:ss [GMT]');
         let message;
         if (typeof data === 'string') {
-            message = data;
+            message = `${data}`;
         } else {
-            message = data.message;
+            message = `${data.message}`;
```

Comment:
@aaron-binary is there any case we have this object?

Reply:
Whenever a user passes a message with an `undefine`d variable for example.

Reply:
@aaron-binary must check if message is null or undefind dont add it to array